src/main/java/HomePage.java

```diff
@@ -3,16 +3,16 @@
 
 public class HomePage {
 
-
-    private final WebDriver driver;
+    private WebDriver driver = DriverBuilder.INSTANCE.getDriver();
 
     public HomePage(WebDriver driver) {
+        Config.loadProperties("HomePage.properties");
```

Comment:
We will not use properties for describing page elements locators. Let's use FindBy annotation instead and PageFactory.initElements(...) for instantiating page object